plugins/confirm.cpp

```diff
@@ -16,27 +16,28 @@
 #include "uicommon.h"
 
 #include "df/building_tradedepotst.h"
+#include "df/gamest.h"
 #include "df/general_ref.h"
 #include "df/general_ref_contained_in_itemst.h"
 #include "df/interfacest.h"
 #include "df/viewscreen_dwarfmodest.h"
-#include "df/viewscreen_jobmanagementst.h"
-#include "df/viewscreen_justicest.h"
-#include "df/viewscreen_layer_militaryst.h"
-#include "df/viewscreen_locationsst.h"
-#include "df/viewscreen_tradegoodsst.h"
+//#include "df/viewscreen_jobmanagementst.h"
+//#include "df/viewscreen_justicest.h"
+//#include "df/viewscreen_layer_militaryst.h"
+//#include "df/viewscreen_locationsst.h"
+//#include "df/viewscreen_tradegoodsst.h"
 
 using namespace DFHack;
 using namespace df::enums;
 using std::map;
 using std::queue;
 using std::string;
 using std::vector;
+using df::global::game;
```

Comment:
You should use `REQUIRE_GLOBAL(game)` instead. That ensures that the global address is available when loading the plugin.